src/python/visclaw/gaugetools.py

```diff
@@ -102,6 +102,8 @@ def __repr__(self):
             output = "%4i" % self.number
             output = " ".join((output,"%19.10e" % self.location[0]))
```

Comment:
This could probably be cleaned up using either a list comprehension or a for loop.